src/Features/Core/Portable/PullMemberUp/QuickAction/InterfacePullerWithQuickAction.cs

```diff
@@ -8,7 +8,7 @@ internal class InterfacePullerWithQuickAction : AbstractMemberPullerWithQuickAct
     {
         private static InterfacePullerWithQuickAction s_puller;
 
-        internal static InterfacePullerWithQuickAction GetInstance
+        internal static InterfacePullerWithQuickAction Instance
```

Comment:
no need for these properties.  The static will be created on demand the first time they are needed anyways.